pkg/cmd/server/server.go

```diff
@@ -379,10 +410,10 @@ func (s *server) loadConfig() (*api.Config, error) {
 }
 
 const (
-	defaultGCSyncPeriod              = 60 * time.Minute
 	defaultBackupSyncPeriod          = 60 * time.Minute
-	defaultScheduleSyncPeriod        = time.Minute
 	defaultPodVolumeOperationTimeout = 60 * time.Minute
+	gcSyncPeriod                     = 60 * time.Minute
```

Comment:
since `gcSyncPeriod` and `scheduleSyncPeriod` aren't configurable, let's move them into `pkg/controller` (first in `gc_controller.go` and second in `schedule_controller.go`) and remove them from the constructors for those controllers.